### PR TITLE
Repackage dd-trace-dotnet to ls-trace-dotnet

### PR DESCRIPTION
**Pull request by MikeGoldsmith**
- update README
- increase GitVersion to v5
- increase ubuntu 14.04 to 16.04
- update nuget packages and apm's to use LightStep prefix

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -9,14 +9,14 @@
 
     <!-- NuGet packages -->
     <IsPackable>true</IsPackable>
-    <PackageIconUrl>https://github.com/DataDog/dd-trace-dotnet/raw/master/datadog-logo-64x64.png</PackageIconUrl>
-    <PackageProjectUrl>https://docs.datadoghq.com/tracing/setup/dotnet/</PackageProjectUrl>
+    <PackageIconUrl>https://github.com/LightStep/lightstep-tracer-csharp/raw/master/ls128x128.png</PackageIconUrl>
+    <PackageProjectUrl>hhttps://docs.lightstep.com/docs/net-auto-instrumentation</PackageProjectUrl>
```
Comment (codeboten) [inline src/Directory.Build.props]:
```suggestion
    <PackageProjectUrl>https://docs.lightstep.com/docs/net-auto-instrumentation</PackageProjectUrl>
```

File: src/Directory.Build.props [hunk 2]
```diff
@@ -9,14 +9,14 @@
...
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageReleaseNotes>See release notes at https://github.com/DataDog/dd-trace-dotnet/releases</PackageReleaseNotes>
-    <PackageTags>Datadog;APM;tracing;profiling;instrumentation</PackageTags>
+    <PackageReleaseNotes>See release notes at https://github.com/LightStep/ls-trace-dotnet/releases</PackageReleaseNotes>
```
Comment (codeboten) [inline src/Directory.Build.props]:
```suggestion
    <PackageReleaseNotes>See release notes at https://github.com/lightstep/ls-trace-dotnet/releases</PackageReleaseNotes>
```

File: src/Directory.Build.props [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
+    <PackageTags>lightstep;APM;tracing;profiling;instrumentation</PackageTags>
     <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/DataDog/dd-trace-dotnet.git</RepositoryUrl>
+    <RepositoryUrl>https://github.com/LightStep/ls-trace-dotnet.git</RepositoryUrl>
```
Comment (codeboten) [inline src/Directory.Build.props]:
```suggestion
    <RepositoryUrl>https://github.com/lightstep/ls-trace-dotnet.git</RepositoryUrl>
```

File: src/Datadog.Trace.OpenTracing/Datadog.Trace.OpenTracing.csproj [hunk 4]
```diff
@@ -2,8 +2,9 @@
 
   <PropertyGroup>
     <!-- NuGet -->
+    <PackageId>LightStep.Trace.OpenTracing</PackageId>
     <Version>1.9.0</Version>
-    <Title>Datadog APM - OpenTracing</Title>
+    <Title>LightStep APM - OpenTracing fork</Title>
```
Comment (codeboten) [inline src/Datadog.Trace.OpenTracing/Datadog.Trace.OpenTracing.csproj]:
```suggestion
    <Title>LightStep's Datadog APM fork - OpenTracing</Title>
```

File: src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj [hunk 5]
```diff
@@ -4,8 +4,9 @@
     <RootNamespace>Datadog.Trace.ClrProfiler</RootNamespace>
 
     <!-- NuGet -->
+    <PackageId>LightStep.Trace.ClrProfiler.Managed</PackageId>
     <Version>1.9.0</Version>
-    <Title>Datadog APM - ClrProfiler</Title>
+    <Title>LightStep Datadog APM - ClrProfiler fork</Title>
```
Comment (codeboten) [inline src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj]:
```suggestion
    <Title>LightStep's Datadog APM fork - ClrProfiler</Title>
```

File: reproductions/AutomapperTest/Dockerfile [hunk 6]
```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/dotnet/core/runtime:2.1-stretch-slim AS base
 ARG TRACER_VERSION=1.9.0
 RUN mkdir -p /opt/datadog
 RUN mkdir -p /var/log/datadog
-RUN curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v$TRACER_VERSION/datadog-dotnet-apm-$TRACER_VERSION.tar.gz | tar xzf - -C /opt/datadog
+RUN curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v$TRACER_VERSION/lightstep-dotnet-apm-$TRACER_VERSION.tar.gz | tar xzf - -C /opt/datadog
```
Comment (codeboten) [inline reproductions/AutomapperTest/Dockerfile]:
```suggestion
RUN curl -L https://github.com/lightstep/ls-trace-dotnet/releases/download/v$TRACER_VERSION/lightstep-dotnet-apm-$TRACER_VERSION.tar.gz | tar xzf - -C /opt/datadog
```

File: docs/README.md [hunk 7]
```diff
@@ -1,123 +1,33 @@
...
-  - Requires Windows 10 (1607 Anniversary Update, Build 14393 or newer)
-
-Microsoft provides [evaluation developer VMs](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines) with Windows 10 and Visual Studio pre-installed.
-
-### Building from a command line
-
-From a _Developer Command Prompt for VS 2019_:
+Windows and Linux Installers|[See releases](https://github.com/LightStep/ls-trace-dotnet/releases)
+`LightStep.Trace`|[![Datadog.Trace](https://img.shields.io/nuget/vpre/LightStep.Trace.svg)](https://www.nuget.org/packages/LightStep.Trace)
+`LightStep.Trace.ClrProfiler.Managed`|[![LightStep.Trace.ClrProfiler.Managed](https://img.shields.io/nuget/vpre/LightStep.Trace.ClrProfiler.Managed.svg)](https://www.nuget.org/packages/LightStep.Trace.ClrProfiler.Managed)
```
Comment (codeboten) [inline docs/README.md:17]: Is `Lighstep.Trace.OpenTracing` missing here?

Comment (MikeGoldsmith) [reply docs/README.md:17]: It's not actually used, I'm not entirely sure why they have that project in the repo.

Comment (codeboten) [reply docs/README.md:17]: Ah k.

**Review by codeboten — APPROVED** [review]

**Pull request merged**